### ISSUE TITLE
chart: Instructions for uninstall Longhorn needs to be updated

### ISSUE THREAD

**Issue by yasker** [rancher/longhorn#261]
To include remove Longhorn related PV/PVC in the Kubernetes first, then follow the latest instruction at https://github.com/rancher/longhorn

**Comment by rootwuj** [comment]
I used the catalog apps to successfully create a longhorn 0.3.0, and attach volume, then detach volume, and delete the PV/PVC in the k8s cluster.
Then follow the latest instructions to uninstall longhorn: I executed "Clean up volume and related resources" /"Clean up engine images and nodes", did not execute delete yaml. 
Finally, I removed the longhorn instance from UI, but the CSI-related pod was not deleted.
Please check if the chart has not deleted this part.
```
root@ip-172-31-10-193:~/# kubectl -n longhorn-system get pod
NAME                        READY     STATUS    RESTARTS   AGE
csi-attacher-0              1/1       Running   0          17m
csi-provisioner-0           1/1       Running   0          17m
longhorn-csi-plugin-c7n9k   2/2       Running   0          17m
longhorn-csi-plugin-srfsk   2/2       Running   0          17m
longhorn-csi-plugin-t7xq2   2/2       Running   0          17m
root@ip-172-31-10-193:~/#
```

**Comment by LLParse** [comment]
I've reproduced the issue where deleting the `longhorn-driver-deployer` Deployment gracefully doesn't delete the underlying driver. We'll have to document some manual steps for 0.3.0 and investigate what the underlying issue is.

**Comment by yasker** [comment]
Yeah, let's document the step in the Longhorn repo (as /docs/charts.md) and copy (and link) it to the chart.

Also, please file an issue for investigating this.
